### PR TITLE
Update setup.html with environment setup

### PR DESCRIPTION
**Pull request by derekngo**
- Added description on how to set up environment with IntelliJ
- Screenshot of change here: [screenshot](http://puu.sh/hsNNF/ba9e2f9b8d.png)
- Summary of content added: [gist of content added](https://gist.github.com/derekngo/a9f7bf4444f8c26d3515)

### CODE REVIEW

File: includes/setup.html [hunk 1]
```diff
@@ -240,6 +241,125 @@ <h2 >Launching ios-driver</h2>
...
+                <li>It should default to /Library/Java/JavaVirtualMachines/jdk1.x.x_xx.jdk/Contents/Home (ex: jdk1.7.0.51.jdk)</li>
+            </ul>
+        If a notification pops up with auto-import, select ‘enable auto-import’
+
+    <h3>Once your environment is set up:</h3>
+    <h4>Creating a simple WebView test (in IntelliJ)</h4>
+
+        File > New Project
+            <ul>
+                <li>Select Maven on the left hand side & Under Project SDK: ‘1.7 (java version)’ or ‘1.8 (java version)’</li>
```
Comment (mmccartney) [inline includes/setup.html]: probably need `&amp;` here

Comment (derekngo) [inline includes/setup.html]: fixed and amended commit

**Comment by mmccartney** [comment]
Looks like good detailed documentation.  Any reason not to include this @mach6 ?

**Comment by mach6** [comment]
:+1: Looks okay to me. Though I'd guess that @freynaud would prefer this be merged to `gh-pages-dev` instead.

**Pull request closed without merging**